src/main/java/teammates/storage/entity/Student.java

```diff
@@ -93,6 +111,27 @@ public Student(String email, String name, String googleId, String comments, Stri
         this.setCourseId(courseId);
         this.setTeamName(teamName);
         this.setSectionName(sectionName);
+        
+        this.setCreated(new Date());
+    }
+    
+    public Date getCreated() {
+        return created;
+    }
+    
+    public void setCreated(Date created) {
+        this.created = created;
+        setLastUpdate(created);
+    }
+    
+    public Date getLastUpdate() {
+        return lastUpdate;
+    }
+    
+    public void setLastUpdate(Date lastUpdate) {
+        if (keepUpdateTimestamp == false) {
```

Comment:
Use `if (!keepUpdateTimestamp)`? Or did you use this format to make it hard to miss?